homeassistant/components/device_tracker/__init__.py

```diff
@@ -89,7 +90,7 @@ def is_on(hass: HomeAssistantType, entity_id: str=None):
 def see(hass: HomeAssistantType, mac: str=None, dev_id: str=None,
         host_name: str=None, location_name: str=None,
         gps: GPSType=None, gps_accuracy=None,
-        battery=None):  # pylint: disable=too-many-arguments
+        battery=None, attrs: dict=None):  # pylint: disable=too-many-arguments
```

Comment:
Keep the name consistent `attributes`. Allows for passing as attributes={} in all see functions